### ISSUE TITLE
Doc mismatch for HEADLESS_FRONTEND_URLS setting

### ISSUE THREAD

**Issue by thclark** [pennersr/django-allauth#3858]
Hi, and firstly a HUGE THANKYOU for the headless API. It's taking so much misery out of life right now! I switched from nightmarish auth0 to allauth, and sponsored the project (as my company, Octue), because of the headless feature (better the money go to you than Okta!).

> Sidenote - in your sponsorship options you could put an entry that's the same as the basic tier in auth0 and highlight all the benefits compared to using auth0 ;)

Next, I'm going through the headless docs and I noticed that [this page](https://docs.allauth.org/en/latest/headless/installation.html) contains:
```py
HEADLESS_FRONTEND_URLS = {
    # ...
    "account_signup": "https://app.org/account/signup",
}
```

Whilst [this page](https://docs.allauth.org/en/latest/headless/configuration.html) contains:
```py
HEADLESS_FRONTEND_URLS = {
    # ...
    "account_signup_url": "https://app.project.org/account/signup",
    # ...
}
```

So for clarification, is it  `account_signup` or `account_signup_url`?

**Comment by pennersr** [comment]
Thank you! Really glad to see it being picked up...  as for the URL, the documentation is wrong, fixed here: 72026c94

PS: Nice way of looking at the sponsorship tiers, though not sure I want to name/promote Auth0 in there... in any case, thanks!

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/3858
